### ISSUE TITLE
`-static-msg` raise an error on static string concatenation

### ISSUE THREAD

**Issue by gnuletik** [go-simpler/sloglint#79]
The following code:

```go
logger.Warn(
    "this is a really long message, that " +
    "requires to split the line",
)
```

Raises the following error in `golangci-lint@v2.0.2` (which uses `simpler.org/sloglint@v0.9.0`):
```
message should be a string literal or a constant (sloglint)
```

The message is constant even though it is built with string concatenation to split long lines.

Related to https://github.com/go-simpler/sloglint/issues/17
